scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScaladocParser.scala

```diff
@@ -148,21 +157,36 @@ object ScaladocParser {
     }
   }
 
-  private def listBlockParser[_: P](minIndent: Int = 1): P[ListBlock] = P {
+  private def listItemParser[_: P](indent: Int): P[ListItem] = {
+    def innerBody =
+      (listBlockParser(indent) |
+        mdCodeBlockParser(indent) |
+        tableParser(indent) |
+        textParser(indent)).rep(0)
+
+    (textParser(indent) ~ innerBody)
+      .map { case (beginning, other) => ListItem(beginning +: other) }
+  }
+
+  private def listBlockParser[_: P]: P[ListBlock] = listBlockParser(0)
```

Comment:
turns out indent can't be 0; the outermost list has to have at least one (hence `minIndent: Int = 1` above).

Reply:
Oh, yep - found it by attached link to scaladoc spec. On the other hand, it's allowed in scala3.

I think in general it would be better to allow it. I think in general people don't know how comment is rendered in scaladoc and can't notice that they missed space before list item. I assume there will be more unexpected rewrites rather that invalid list detection.

Reply:
current approach is requiring one space (not "one more space than the rest of text", as documentation describes), but it doesn't allow having absolutely no spaces at all. problem with that is, the list marker can really be a continuation of the previous line ("the default value of this parameter is <nl> 1." or "it can be 2 <nl> - or 3 - in some cases"). so, I wouldn't relax.

Reply:
hmm, I've just realized that without this rule might become unpredictable. There is the case when scalfmt might move `-` to the beginning of next line and turn it into a list. I agree, let's keep this rule ignoring scala3 behavior